CUJExecutor/app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.firstapp">
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.KILL_BACKGROUND_PROCESSES" />
```

Comment:
Why does the testapp need this permission?

Reply:
Ahh it probably doesn't (this is from a while back when I was trying to figure out a better way to force-stop apps)